CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.1.4 (Apr 6, 2022)
```

Comment:
I'd make this 0.2.0. Technically this is a breaking change but we're pre v1 so I think 0.2.0 is the most correct. Should this also be covered in the README?

Reply:
I don't think it's a breaking change. worst case a use would have handled the undefined scenario (that would never happen) and now it is just redundant to handle it.

Reply:
There is nothing in the readme about using the parse/serialize methods directly (which the types are used for).
But I could add something in there 👍

Reply:
yea it always threw. all I did was change the type to reflect that.